### ISSUE TITLE
Go 1.5 Support

### ISSUE THREAD

**Issue by icdev** [Shippable/support#1837]
I might be missing something, but I'm getting this following error when trying to build using go 1.5.  (https://golang.org/doc/go1.5) 

``` gvm install go1.5
Updating Go source...
ERROR: Unrecognized Go version```

My shippable configuration has the following 
```
language: go

go:
  - 1.5
```
Any help would be appreciated.

**Comment by garthbrydon** [comment]
we've got the same problem

**Comment by jdextraze** [comment]
Go 1.5 is now written in go. So to fix this you need to install go 1.4 and then compile go 1.5. See more info here: https://github.com/moovweb/gvm/issues/155

**Comment by manishas** [comment]
go 1.5 is available in our official images in the [drydock repo on Docker Hub](https://hub.docker.com/u/drydock/)
